contribute/release-guide.md

```diff
@@ -0,0 +1,500 @@
+---
+layout: default
+title: "Beam Release Guide"
+permalink: /contribute/release-guide/
+---
+
+# Apache Beam Release Guide
+
+* TOC
+{:toc}
+
+## Introduction
+
+The Apache Beam project periodically declares and publishes releases. A release is one or more packages of the project artifact(s) that are approved for general public distribution and use. They may come with various degrees of caveat regarding their perceived quality and potential for change, such as “alpha”, “beta”, “incubating”, “stable”, etc.
+
+The Beam community treats releases with great importance. They are a public face of the project and most users interact with the project only through the releases. Releases are signed off by the entire Beam community in a public vote.
+
+Each release is executed by a *Release Manager*, who is selected among the [Beam committers](http://beam.incubator.apache.org/project/team). This document describes the process that the Release Manager follows to perform a release. Any changes to this process should be discussed and adopted on the [dev@ mailing list](http://beam.incubator.apache.org/use/mailing-lists).
+
+Please remember that publishing software has legal consequences. This guide complements the foundation-wide [Product Release Policy](http://www.apache.org/dev/release.html) and [Release Distribution Policy](http://www.apache.org/dev/release-distribution).
+
+## Overview
+
+![Alt text]({{ "/images/release-guide-1.png" | prepend: site.baseurl }} "Release Process"){:width="100%"}
+
+The release process consists of several steps:
+
+1. Decide to release
+1. Prepare for the release
+1. Build a release candidate
+1. Vote on the release candidate
+1. If necessary, fix any issues and go back to step 3.
+1. Finalize the release
+1. Promote the release
+
+**********
+
+## Decide to release
+
+Deciding to release and selecting a Release Manager is the first step of the release process. This is a consensus-based decision of the entire community.
+
+Anybody can propose a release on the dev@ mailing list, giving a solid argument and nominating a committer as the Release Manager (including themselves). There’s no formal process, no vote requirements, and no timing requirements. Any objections should be resolved by consensus before starting the release.
+
+In general, the community prefers to have a rotating set of 3-5 Release Managers. Keeping a small core set of managers allows enough people to build expertise in this area and improve processes over time, without Release Managers needing to re-learn the processes for each release. That said, if you are a committer interested in serving the community in this way, please reach out to us on the dev@ mailing list.
```

Comment:
would suggest to have this: "please reach out to us on the dev@ mailing list."
To "please reach out to the community on the dev@ mailing list" unless there is a specific list for committers / PMC members